core/components/molecules/placeholder/__stories__/skeletonLoaders.story.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { PlaceholderParagraph, Placeholder, Card } from '@/index';
+
+export const skeletonLoaders = () => {
+
+  return (
+    <div>
+      <Card>
+        <div >
+          <Placeholder className="ml-6 mr-6 mb-5 mt-5" round={true} imageSize="large">
+            <PlaceholderParagraph  length="large" size="m" />
+            <PlaceholderParagraph size="m"/>
+          </Placeholder>
+        </div>
+        <div  style={{ borderTop:'1px solid #e5e5e5' }}>
```

Comment:
We have a css variable corresponding to `'1px solid #e5e5e5'`. Please use that variable.